### DISCUSSION TITLE
Release schedule

### DISCUSSION THREAD

**Discussion by andylaw** [Eugeny/tabby#9938] [General]
@Eugeny - do you have a release schedule planned/in mind, or is it all _ad hoc_?

**Comment by Eugeny** [comment]
No, it's mostly ad-hoc

**Reply by andylaw** [reply]
What's the likelihood of a release in the next few weeks?

To be honest, I'm fishing for an update on #9859 - I want to encourage our local software admin people to make Tabby available as a self-install option on our managed Windows and Mac systems, but the lack of AppleScript capability is a show-stopper for me. I really don't want to ask them to make one version available and then have to ask them a few weeks later to update it - that will burn all my credit.

**Reply by Eugeny** [reply]
Not gonna lie - I have no idea how I missed that PR 😳

**Reply by andylaw** [reply]
No worries. Thanks for merging
